tests/test_blinds.py

```diff
@@ -24,7 +35,26 @@ def test_device_info_properties(device: Device) -> None:
     assert info.battery_level == 77
 
 
+@pytest.mark.parametrize("device", ["blind"], indirect=True)
 def test_current_position(device: Device) -> None:
     """Test blind position."""
     blind = device.blind_control.blinds[0]
     assert blind.current_cover_position == 50
+
+
+@pytest.mark.parametrize("device", ["blind"], indirect=True)
+def test_set_state(device: Device) -> None:
+    """Test set state."""
+    blind_control = device.blind_control
+    assert blind_control is not None
+    command = blind_control.set_state(50)
+    assert command.data == {"15015": [{"5536": 50}]}
+
+
+@pytest.mark.parametrize("device", ["blind"], indirect=True)
+def test_trigger_blind(device: Device) -> None:
+    """Test trigger blind."""
+    blind_control = device.blind_control
+    assert blind_control is not None
+    command = blind_control.trigger_blind()
+    assert command.data == {"15015": [{"5523": True}]}
```

Comment:
I'm suspecting the boolean should be sent as an integer in this command, since that's how we do it in other device types, but I won't change that code without testing it first with a real device.